Dockerfile

```diff
@@ -23,7 +23,7 @@ RUN go build -a -installsuffix cgo -o secrets-provider ./cmd/secrets-provider
 FROM busybox
 
 # =================== MAIN CONTAINER ===================
-FROM scratch as secrets-provider
+FROM alpine:3.8 as secrets-provider
```

Comment:
why do we use 3.8? why not `latest`? why not a newer version?

i'm not saying it's incorrect, just want to verify you put thought into it and chose the best one.

Reply:
good point. i did it for uniformity across projects but 3.11 is available and the local build passes so I will use that one